File: lib/aditem.dart
part of 'mbads.dart';

class AdItem{
  /*
  * 代码位
  * */
  late String codeNo;

  /*
  * ios代码位(可以为空,空的时候用codeNo)
  * */
  String iosCodeNo = "";
  /*
  *  平台
  *   1:穿山甲
  * */
  late int plat;


  AdItem(this.codeNo, this.iosCodeNo, this.plat);

  Map toJson() {
    Map map = {};
    map["codeNo"] = codeNo;
    map["iosCodeNo"] = iosCodeNo;
    map["plat"] = plat;
    return map;
  }

}

File: android/src/main/java/com/mb/mbads/model/AdItem.java
package com.mb.mbads.model;

import java.io.Serializable;

public class AdItem implements Serializable {
    private String codeNo;
    private int plat;

    public String getCodeNo() {
        return codeNo;
    }

    public void setCodeNo(String codeNo) {
        this.codeNo = codeNo;
    }

    public int getPlat() {
        return plat;
    }

    public void setPlat(int plat) {
        this.plat = plat;
    }
}
